infra/base-images/base-builder/precompile_centipede

```diff
@@ -32,6 +32,15 @@ bazel --bazelrc=/tmp/centipede.bazelrc build -c opt :all
 # The weak symbols provides fake implementations for intermediate binaries.
 $CXX "$SRC/centipede/weak_sancov_stubs.cc" -c -o "$SRC/centipede/weak.o"
 
+# Centipede's ld, which always appends weak.o as a parameter.
+export CENTIPEDE_BIN="$SRC/centipede/bin"
+mkdir -p $CENTIPEDE_BIN
+cat << EOF > ${CENTIPEDE_BIN}/ld
+#!/bin/bash
+/usr/bin/ld \$@ -ldl -lrt -lpthread -lc++ /src/centipede/weak.o
+EOF
+chmod 777 ${CENTIPEDE_BIN}/ld
```

Comment:
Are we setting this to be the linker somewhere?

Reply:
[Line 37-39 in `compile_centipede`](https://github.com/google/oss-fuzz/blob/38d53f21e356b384c06d6490b66245ea0ff55056/infra/base-images/base-builder/compile_centipede#L39).

Reply:
Ah got it. Do you mind giving me an hour to test https://github.com/google/oss-fuzz/pull/9300? I have a hunch most of the major fixes have been pushed and I think this PR has some unecessary complexity.

Reply:
Sure, I am working on https://github.com/google/fuzzbench/pull/1632 now.